.github/workflows/CI.yaml

```diff
@@ -70,10 +70,35 @@ jobs:
         pytest -v --cov=membrane_curvature --cov-report=xml --color=yes membrane_curvature/tests/
 
     - name: CodeCov
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v2
       with:
         file: ./coverage.xml
         flags: unittests
         name: codecov-${{ matrix.os }}-py${{ matrix.python-version }}
         fail_ci_if_error: True
 
+  latest_python:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        latest-python : ["3.11.0-rc.2"]
+    
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.latest-python }}
+
+    - name:  pip install membrane-curvature
+      run: |
+        pip install git+https://github.com/MDAnalysis/mdanalysis.git@develop#subdirectory=package
```

Comment:
The CI job seems to be doing the right thing at least, and a bleeding edge job like this probably makes sense anyway.

I think you should also be able to override any restrictions in `pyproject.toml` using `pip install --no-build-isolation ..` and providing the deps manually. If you wanted to use the stable source release of MDAnalysis that might be an option, but what you have here now probably makes more sense.